src/snowcli/cli/snowpark/commands.py

```diff
@@ -0,0 +1,385 @@
+from __future__ import annotations
+
+import logging
+import os
+from pathlib import Path
+from tempfile import TemporaryDirectory
+from typing import List, Dict
+
+import typer
+from click import ClickException
+from snowflake.connector import ProgrammingError
+
+from snowcli import utils
+from snowcli.cli.common.decorators import global_options_with_connection, global_options
+from snowcli.cli.common.flags import (
+    DEFAULT_CONTEXT_SETTINGS,
+    identifier_argument,
+    execution_identifier_argument,
+)
+from snowcli.cli.common.project_initialisation import add_init_command
+from snowcli.cli.constants import DEPLOYMENT_STAGE, ObjectType, SnowparkObjectType
+from snowcli.cli.project.definition_manager import DefinitionManager
+from snowcli.cli.snowpark.common import (
+    remove_parameter_names,
+    check_if_replace_is_required,
+    build_udf_sproc_identifier,
+)
+from snowcli.cli.snowpark.manager import ProcedureManager, FunctionManager
+
+from snowcli.cli.snowpark_shared import (
+    CheckAnacondaForPyPiDependencies,
+    PackageNativeLibrariesOption,
+    PyPiDownloadOption,
+    snowpark_package,
+)
+from snowcli.cli.object.stage.manager import StageManager
+from snowcli.exception import ObjectAlreadyExistsError
+from snowcli.output.decorators import with_output
+from snowcli.output.types import (
+    MessageResult,
+    CommandResult,
+    SingleQueryResult,
+    QueryResult,
+    CollectionResult,
+)
+from snowcli.utils import (
+    get_snowflake_packages,
+)
+
+
+log = logging.getLogger(__name__)
+
+app = typer.Typer(
+    name="snowpark",
+    context_settings=DEFAULT_CONTEXT_SETTINGS,
+    help="Manage functions and procedures.",
```

Comment:
Decide on one order `functions and/or procedures` or `procedures and/or functions`

Reply:
I think there is more `procedures and/or functions`

Reply:
Fixed